packages/editor/src/components/block-draggable/index.js

```diff
@@ -4,13 +4,12 @@
 import { Draggable } from '@wordpress/components';
 import { withSelect } from '@wordpress/data';
 
-const BlockDraggable = ( { children, clientId, rootClientId, blockElementId, index, layout, onDragStart, onDragEnd } ) => {
+const BlockDraggable = ( { children, clientId, rootClientId, blockElementId, index, onDragStart, onDragEnd } ) => {
 	const transferData = {
 		type: 'block',
-		fromIndex: index,
-		rootClientId,
-		clientId,
-		layout,
+		srcIndex: index,
```

Comment:
Improve nomenclature, as it was inconsistent (`fromIndex` but not `fromClientId`?), especially when this data was used in the `BlockDropZone`.